src/app/CommandSender.cpp

```diff
@@ -91,6 +91,10 @@ CHIP_ERROR CommandSender::AllocateBuffer()
         mInvokeRequestBuilder.CreateInvokeRequests();
         ReturnErrorOnFailure(mInvokeRequestBuilder.GetError());
 
+        auto * tlvWriter = mInvokeRequestBuilder.GetWriter();
+        ReturnErrorOnFailure(tlvWriter->ReserveBuffer(mInvokeRequestBuilder.GetInvokeRequests().GetSizeToEndInvokeRequests()));
```

Comment:
Again, would be best to use the writer from the thing that will end up touching that writer.  Followup fine.